.php_cs

```diff
@@ -17,18 +17,181 @@ $finder = PhpCsFixer\Finder::create()
+        'phpdoc_no_alias_tag'                         => ['property-read' => 'property', 'property-write' => 'property', 'type' => 'var'],
+        'phpdoc_no_empty_return'                      => false,
+        'phpdoc_no_package'                           => true,
+        'phpdoc_no_useless_inheritdoc'                => true,
+        'phpdoc_order'                                => true,
+        'phpdoc_return_self_reference'                => true,
+        'phpdoc_scalar'                               => true,
+        'phpdoc_separation'                           => true,
+        'phpdoc_single_line_var_spacing'              => true,
+        'phpdoc_summary'                              => false,
+        'phpdoc_to_comment'                           => true,
+        'phpdoc_trim'                                 => true,
+        'phpdoc_types'                                => true,
+        'phpdoc_types_order'                          => ['null_adjustment' => 'always_last', 'sort_algorithm' => 'alpha'],
+        'phpdoc_var_without_name'                     => true,
+        'pow_to_exponentiation'                       => false,
+        'protected_to_private'                        => true,
+        'psr0'                                        => true,
+        'psr4'                                        => true,
+        'random_api_migration'                        => true,
+        'return_type_declaration'                     => ['space_before' => 'none'],
+        'self_accessor'                               => true,
+        'semicolon_after_instruction'                 => true,
+        'short_scalar_cast'                           => true,
+        'silenced_deprecation_error'                  => true,
+        'simplified_null_return'                      => false,
+        'single_blank_line_at_eof'                    => true,
+        'single_blank_line_before_namespace'          => false,
+        'single_class_element_per_statement'          => ['elements' => ['const', 'property']],
+        'single_import_per_statement'                 => true,
+        'single_line_after_imports'                   => true,
+        'single_line_comment_style'                   => ['comment_types' => ['asterisk', 'hash']],
+        'single_quote'                                => true,
+        'space_after_semicolon'                       => ['remove_in_empty_for_expressions' => true],
+        'standardize_not_equals'                      => true,
+        'static_lambda'                               => true,
+        'strict_comparison'                           => true,
+        'strict_param'                                => false,
+        'switch_case_semicolon_to_colon'              => true,
+        'switch_case_space'                           => true,
+        'ternary_operator_spaces'                     => true,
+        'ternary_to_null_coalescing'                  => false,
```

Comment:
This is opposite of what's in PHP70Migration (and I actually prefer this one to be `true`).

Reply:
fixed